go.mod

```diff
@@ -151,13 +151,12 @@ require (
 	go.uber.org/multierr v1.6.0 // indirect
 	go.uber.org/zap v1.19.1 // indirect
 	golang.org/x/crypto v0.0.0-20220214200702-86341886e292 // indirect
-	golang.org/x/mod v0.6.0-dev.0.20220106191415-9b9b3d81d5e3 // indirect
-	golang.org/x/net v0.0.0-20220127200216-cd36cc0744dd // indirect
+	golang.org/x/mod v0.6.0-dev.0.20220419223038-86c51ed26bb4 // indirect
+	golang.org/x/net v0.1.1-0.20221027164007-c63010009c80 // indirect
```

Comment:
The latest stable version of [golang.org/x/net](https://pkg.go.dev/golang.org/x/net@v0.1.0) is "v0.1.0". It seems you used unpublished version?

See commit "a2d827a" from <https://cs.opensource.google/go/x/net/+/master:;bpv=1>

> Update golang.org/x dependencies to their latest tagged versions.
Once this CL is submitted, and post-submit testing succeeds on all
first-class ports across all supported Go versions, this repository
will be tagged with its next minor version.